egs/wsj/s5/utils/combine_data.sh

```diff
@@ -30,7 +30,7 @@ shift;
 
 first_src=$1;
 
-rm -r $dest 2>/dev/null
+rm -rf $dest 2>/dev/null
```

Comment:
Rather than adding `-f`, which can be a little dangerous, overriding user preferences, I prefer to add "|| true" to such statements.  
Note: this statement would only have caused a problem if you had done
`set -e`
for instance, in your .bash_profile.

Reply:
Thanks! I'll add only `|| true`

Reply:
I've checked with `set -e`, `set -u`, `set -o pipefail`